### ISSUE TITLE
Missing AdoptOpenJDK releases where there are >20 for a majorVersion/arch/os combination

### ISSUE THREAD

**Issue by chadlwilson** [halcyon/asdf-java#90]
It seems the V3 API has [a hidden/undocumented max](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/811367c50ece18b5b94728ee6d50f3568da6d7c2/adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/AssetsResource.kt#L275) `page_size` of `20`. Despite this plugin setting the page size to `100` this means release lists are getting cropped; as the plugin doesn't have the ability to iterate through pages of `feature_releases` API calls.

Current this means that the `mac` `x64` adoptopen jdk releases of `11.0.7+10` are unable to be located.

I've asked about this [here](https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/184#issuecomment-622671832), but if the behaviour is intentional we may need to find a way to either

- support paging of releases
- sort `DESC` so we always get the most recent 20 releases rather than the oldest (?)

**Comment by Nezteb** [comment]
Bumping this.

```
$ asdf list-all java
amazon-corretto-11.0.3.7.1
amazon-corretto-11.0.4.11.1
amazon-corretto-11.0.5.10.1
amazon-corretto-11.0.5.10.2
amazon-corretto-11.0.6.10.1
amazon-corretto-11.0.6.10.1-2
amazon-corretto-11.0.7.10.1
amazon-corretto-8.212.04.2
amazon-corretto-8.232.09.1
amazon-corretto-8.232.09.2
amazon-corretto-8.242.07.1
amazon-corretto-8.242.08.1
amazon-corretto-8.252.09.1
azul-zulu-11.31.11-jdk11.0.3
azul-zulu-11.35.13-jdk11.0.5
azul-zulu-11.35.15-jdk11.0.5
azul-zulu-11.37.17-jdk11.0.6
azul-zulu-11.39.15-jdk11.0.7
azul-zulu-12.2.3-jdk12.0.1
azul-zulu-13.27.9-jdk13
azul-zulu-13.28.11-jdk13.0.1
azul-zulu-13.29.9-jdk13.0.2
azul-zulu-14.27.1-jdk14
azul-zulu-7.29.0.5-jdk7.0.222
azul-zulu-7.34.0.5-jdk7.0.242
azul-zulu-7.36.0.5-jdk7.0.252
azul-zulu-8.42.0.21-jdk8.0.232
azul-zulu-8.42.0.23-jdk8.0.232
azul-zulu-8.44.0.11-jdk8.0.242
azul-zulu-8.44.0.9-jdk8.0.242
azul-zulu-8.46.0.19-jdk8.0.252
azul-zulufx-11.35.15-jdk11.0.5
azul-zulufx-11.37.19-jdk11.0.6
azul-zulufx-8.42.0.23-jdk8.0.232
azul-zulufx-8.44.0.13-jdk8.0.242
azul-zulufx-8.46.0.19-jdk8.0.252
```

FYI @halcyon @joschi.

**Comment by chadlwilson** [comment]
AdoptOpenJdk have now documented and confirmed the max page size of `20` in the issue linked above so it seems the pagination will probably need to be handled in the `asdf-java` plugin.

**Comment by joschi** [comment]
This will be resolved in #87.
```
# asdf list-all java adopt
adoptopenjdk-8.0.181+13
adoptopenjdk-8.0.192+12
adoptopenjdk-8.0.202+8
adoptopenjdk-8.0.202+8.openj9-0.12.0
adoptopenjdk-8.0.202+8.openj9-0.12.1
adoptopenjdk-8.0.212+3
adoptopenjdk-8.0.212+3.openj9-0.14.0
adoptopenjdk-8.0.212+4
adoptopenjdk-8.0.212+4.openj9-0.14.2
adoptopenjdk-8.0.222+10.1
adoptopenjdk-8.0.222+10.1.openj9-0.15.1
adoptopenjdk-8.0.232+9.1
adoptopenjdk-8.0.232+9.1.openj9-0.17.0
adoptopenjdk-8.0.242+8.1
adoptopenjdk-8.0.242+8.1.openj9-0.18.1
adoptopenjdk-8.0.252+9.1
adoptopenjdk-8.0.252+9.1.openj9-0.20.0
adoptopenjdk-9.0.0+181
adoptopenjdk-10.0.2+13.1
adoptopenjdk-11.0.0+28
adoptopenjdk-11.0.1+13
adoptopenjdk-11.0.1+13.openj9-0.11.0
adoptopenjdk-11.0.2+7
adoptopenjdk-11.0.2+9
adoptopenjdk-11.0.2+9.openj9-0.12.0
adoptopenjdk-11.0.2+9.openj9-0.12.1
adoptopenjdk-11.0.3+7
adoptopenjdk-11.0.3+7.openj9-0.14.0
adoptopenjdk-11.0.3+7.openj9-0.14.3
adoptopenjdk-11.0.4+11.1
adoptopenjdk-11.0.4+11.1.openj9-0.15.1
adoptopenjdk-11.0.5+10.1
adoptopenjdk-11.0.5+10.1.openj9-0.17.0
adoptopenjdk-11.0.6+10.1
adoptopenjdk-11.0.6+10.1.openj9-0.18.0
adoptopenjdk-11.0.6+10.1.openj9-0.18.1
adoptopenjdk-11.0.7+10.1
adoptopenjdk-11.0.7+10.1.openj9-0.20.0
adoptopenjdk-12.0.0+33
adoptopenjdk-12.0.0+33.openj9-0.13.0
adoptopenjdk-12.0.1+12
adoptopenjdk-12.0.1+12.openj9-0.14.1
adoptopenjdk-12.0.2+10.1
adoptopenjdk-12.0.2+10.1.openj9-0.15.1
adoptopenjdk-12.0.2+10.3.openj9_0.15.1
adoptopenjdk-13.0.0+33.1
adoptopenjdk-13.0.0+33.1.openj9-0.16.0
adoptopenjdk-13.0.1+9.1
adoptopenjdk-13.0.1+9.1.openj9-0.17.0
adoptopenjdk-13.0.2+8.1
adoptopenjdk-13.0.2+8.1.openj9-0.18.0
adoptopenjdk-14.0.0+36.1
adoptopenjdk-14.0.0+36.1.openj9-0.19.0
adoptopenjdk-14.0.1+7.1
adoptopenjdk-14.0.1+7.1.openj9-0.20.0
adoptopenjdk-large_heap-8.0.202+8.openj9-0.12.1
adoptopenjdk-large_heap-8.0.212+3.openj9-0.14.0
adoptopenjdk-large_heap-8.0.212+4.openj9-0.14.2
adoptopenjdk-large_heap-8.0.222+10.1.openj9-0.15.1
adoptopenjdk-large_heap-8.0.232+9.1.openj9-0.17.0
adoptopenjdk-large_heap-8.0.242+8.1.openj9-0.18.1
adoptopenjdk-large_heap-8.0.252+9.1.openj9-0.20.0
adoptopenjdk-large_heap-11.0.2+9.openj9-0.12.0
adoptopenjdk-large_heap-11.0.2+9.openj9-0.12.1
adoptopenjdk-large_heap-11.0.3+7.openj9-0.14.0
adoptopenjdk-large_heap-11.0.3+7.openj9-0.14.3
adoptopenjdk-large_heap-11.0.4+11.1.openj9-0.15.1
adoptopenjdk-large_heap-11.0.5+10.1.openj9-0.17.0
adoptopenjdk-large_heap-11.0.6+10.1.openj9-0.18.0
adoptopenjdk-large_heap-11.0.6+10.1.openj9-0.18.1
adoptopenjdk-large_heap-11.0.7+10.1.openj9-0.20.0
adoptopenjdk-large_heap-12.0.0+33.openj9-0.13.0
adoptopenjdk-large_heap-12.0.1+12.openj9-0.14.1
adoptopenjdk-large_heap-12.0.2+10.1.openj9-0.15.1
adoptopenjdk-large_heap-13.0.0+33.1.openj9-0.16.0
adoptopenjdk-large_heap-13.0.1+9.1.openj9-0.17.0
adoptopenjdk-large_heap-13.0.2+8.1.openj9-0.18.0
adoptopenjdk-large_heap-14.0.0+36.1.openj9-0.19.0
adoptopenjdk-large_heap-14.0.1+7.1.openj9-0.20.0
```
